packages/vscode/src/plugins/prisma-language-server/index.ts

```diff
@@ -68,23 +94,18 @@ const plugin: PrismaVSCodePlugin = {
 
     const rootPath = workspace.workspaceFolders?.[0].uri.path
     if (rootPath) {
-      watcher = chokidar.watch(
-        path.join(rootPath, '**/node_modules/.prisma/client/index.d.ts'),
-        {
-          // ignore dotfiles (except .prisma) adjusted from chokidar README example
-          ignored: /(^|[\/\\])\.(?!prisma)./,
-          // limits how many levels of subdirectories will be traversed.
-          // Note that `node_modules/.prisma/client/` counts for 3 already
-          // Example
-          // If vs code extension is open in root folder of a project and the path to index.d.ts is
-          // ./server/database/node_modules/.prisma/client/index.d.ts
-          // then the depth is equal to 2 + 3 = 5
-          depth: 9,
-          // When false, only the symlinks themselves will be watched for changes
-          // instead of following the link references and bubbling events through the link's path.
-          followSymlinks: false,
-        },
-      )
+      const isFileWatcherEnabled = workspace
+        .getConfiguration('prisma')
+        .get('fileWatcher')
```

Comment:
Does this default to `true` after update?

Reply:
Should be default true thanks to https://github.com/prisma/language-tools/pull/1001/files#diff-27c6417703030255980b765bde4e51791ce02be1f10ba3aec5452da8cf55d0c5R81
But to double check with a clean state again